src/app/shared/components/field-error/errors.ts

```diff
@@ -12,6 +12,7 @@ export enum ErrorCode {
   min = 'min',
   max = 'max',
   matDatepickerParse = 'matDatepickerParse',
+  locationIllegalCharacters = 'locationIllegalCharacters',
```

Comment:
Trochę dziwne miejsce na to pole, pomiędzy `matDatepickerParse` i `matDatepickerMin`. Ta sama uwaga niżej.

Reply:
to prawda, zbyt dużym pośpiechu rozwiązywałem konflikt